api/src/main/java/io/grpc/ServerCall.java

```diff
@@ -209,6 +209,10 @@ public void setCompression(String compressor) {
     // noop
   }
 
+  public void setOnReadyThreshold(int numBytes) {
```

Comment:
All new public (non-internal) APIs should have `@ExperimentalApi` when added. So here, CallOptions, and stub. You can use the same issue URL for all of them.